docs/platforms/gke/base/use-cases/inference-ref-arch/inference-best-practices.md

```diff
@@ -0,0 +1,193 @@
+    regions using
+    [Storage Transfer Service](https://cloud.google.com/storage-transfer/docs/overview).
+  - Store model files in single-region buckets, and run at least one instance of
+    the Model Downloader for each region to download model files for inference
+    workloads running in that region.
+
+## Online inference with GPUs design choices
+
+This section focuses on the design choices that apply to the
+[Online inference with GPUs](/platforms/gke/base/use-cases/inference-ref-arch/examples/online-inference-gpu/README.md)
+use case. The Online inference with GPUs use case builds on top of the GKE
+Inference reference architecture. The design choices described in this section
+apply to the Online inference with GPUs use case, in addition to the
+[GKE Inference reference architecture design choices](#gke-inference-reference-architecture-design-choices).
+
+The Online inference with GPUs use case design choices cover the following:
+
+- GPU selection
+- GPUs obtainability
+
+### GPU selection
+
+To implement the Online inference with GPUs use case, we selected GPUs by
+considering the following selection criteria:
+
+- Obtainability
+- Cost-effectiveness
+- Performance
+
+For the Online inference with GPUs, we selected the following GPUs:
+
+- **NVIDIA** **L4 GPUs (G2 machines)**: Ideal for cost-effective,
+  high-throughput online inference scenarios, offering a good balance of
+  performance and efficiency.
+- **NVIDIA H100 (A3 machines)**: Selected for demanding inference workloads
+  where maximum throughput and low latency are critical.
+- **NVIDIA H200 (A3 machines)**: Selected for providing unparalleled performance
+  for the most memory-intensive inference workloads.
+
+The following table provides a general overview of model compatibility and
+recommended machine types, for the
+[models that are in scope as examples](#ai-models-in-scope-as-examples):
+
+| GPU  | Compatible Models               | Machine type                                                                                                 | GPU count                                               |
+| ---- | ------------------------------- | ------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------- |
+| L4   | Gemma 3                         | g2-standard-96                                                                                               | 4                                                       |
+| H100 | Gemma 3 Llama 3.3 Llama 4 Scout | a3-highgpu-1g (Gemma 3) a3-highgpu-4g (Gemma 3, Llama 3.3) a3-highgpu-8g (Gemma 3, Llama 3.3, Llama 4 Scout) | 1 (a3-highgpu-1g), 4 (a3-highgpu-4g), 8 (a3-highgpu-8g) |
+| H200 | Gemma 3 Llama 3.3 Llama 4 Scout | a3-ultragpu-8g                                                                                               | 8                                                       |
```

Comment:
For this, how about something more like this:
| GPU  | Compatible Models        | Machine type            | GPU count |
| ---- | ------------------------ | ----------------------- | --------- |
| L4   | Gemma 3 27B IT           | g2-standard-96 (8 GPUs) | 1         |
| H100 | Gemma 3 27B IT           | a3-highgpu-1g (1 GPU)   | 1         |
|      | Llama 3.3 70B IT         | a3-highgpu-4g (4 GPUs)  | 4         |
|      | Llama 4 Scout 17B 16E IT | a3-highgpu-8g (8 GPUs)  | 8         |
| H200 | Gemma 3 27B IT           | a3-ultragpu-8g (8 GPUs) | 1         |
|      | Llama 3.3 70B IT         | a3-ultragpu-8g (8 GPUs) | 4         |
|      | Llama 4 Scout 17B 16E IT | a3-ultragpu-8g (8 GPUs) | 8         |

Reply:
For Gemma 3 do we need a `g2-standard-96` if we are only using 1 GPU?

Reply:
Table refactored as suggested.